tests/types/hexColorCode.spec.ts

```diff
@@ -0,0 +1,74 @@
+import {StrongHexCode, makeHexCode} from '../../src/types/hexColorCode';
```

Comment:
Update the import to reflect the filename which will change to all lowercase.

- [ ] Change import from `hexColorCode` to `hex-color-code`.
- [ ] Change `StrongHexCode` import to `StrongHexColorCode`
- [ ] Change `makeHexCode` import to `makeHexColorCode`